front-end/src/App.js

```diff
@@ -24,7 +25,13 @@ function App() {
   return (
     <Fragment>
       <Header wordHandle={wordHandler} />
-      <main>{showWord && <Word />}</main>
+      <main >{showWord && <Word />}
+      <div style={{marginTop:"6rem"}}>
```

Comment:
remove line 28 , for rest create a parent component which will be called in app.js.
That parent component will contain the infocards.